web/core/src/commonMain/kotlin/androidx/compose/web/css/CSSTypedOM.kt

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2020-2021 JetBrains s.r.o. and respective authors and developers.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+@file:Suppress("UNUSED")
+package androidx.compose.web.css
+
+import kotlin.js.JsName
+
+expect interface CSSStyleValue
+
+open class CSSStyleValueKT : CSSStyleValue {
+    companion object {
+        fun parse(property: String, cssText: String): CSSStyleValue = TODO()
+        fun parseAll(property: String, cssText: String): Array<CSSStyleValue> = TODO()
+    }
+}
+
+expect interface CSSKeywordValue : CSSStyleValue {
+    val value: String
+
+    companion object {
+        operator fun invoke(value: String): CSSKeywordValue
+    }
+}
+
+open class CSSKeywordValueKT(override val value: String) : CSSKeywordValue
+
+// type StylePropertyValue = string | number | CSSStyleValue
+expect interface StylePropertyValue {
+    interface String: StylePropertyValue
+    interface Number: StylePropertyValue
+    interface StyleValue: StylePropertyValue
+    companion object {
+        operator fun invoke(value: kotlin.String): String
+
+        operator fun invoke(value: kotlin.Number): Number
+
+        operator fun invoke(value: CSSStyleValue): StyleValue
+    }
+}
+
+// type CSSNumberish = number | CSSNumericValue
```

Comment:
I suggest to name it `CSSNumeric`  - it's honestly equally understandable and a bit more simple English.

Reply:
I just reused name from TypeScript d.ts typing for it. But CSSNumeric is better, agree)